pandas/tests/test_frame.py

```diff
@@ -10360,6 +10360,15 @@ def test_shift_bool(self):
                        columns=['high', 'low'])
         assert_frame_equal(rs, xp)
 
+    def test_shift_categorical(self):
+        # GH 9416
```

Comment:
so now we need some tests in ``test_categorical`` as well (in fact that should be primary testing location, the others are for confirmation)